test/intl_suggestors/intl_importer_test.dart

```diff
@@ -139,19 +139,17 @@ void main() {
           await testSuggestor(
             input: /*language=dart*/ '''
               import 'package:over_react/over_react.dart';
-
               import 'a/fake_relative_file.dart';
-
+      
               content() => TestProjectIntl.testString;
             ''',
             isExpectedError: (e) =>
                 isUndefinedIntlError(e) || isFakeUriError(e),
             expectedOutput: /*language=dart*/ '''
-              import 'package:over_react/over_react.dart';
               import 'package:test_project/src/intl/test_project_intl.dart';
-
+              import 'package:over_react/over_react.dart';
```

Comment:
The point of the rather complicated _insertLocationForPackageImport is that it tries to put it in alphabetical order. This seems like it's just making it so that for the case of package: imports the new import will always comes first (because it computes the position as if it started with '..') and then changes the test cases so that coming first is expected. I think these should go back to the way they were, and the computation should work on the actual form of the URI that we're going to use.

Reply:
Updated the test cases and